Automattic/jetpack
# Notifications: remove all library dependencies

Following D103999-code and D104607-code (and a few later follow-ups), `wordpress.com` notifications now work without the use of `mustache`, `underscore`, `backbone`, or indeed `jquery`.

This PR correspondingly removes these dependencies from the Jetpack notes module, switching over to the new `notes-common-lite` script.

## Proposed changes:
* Remove all library dependencies from `wordpress.com` notifications

### Other information:

- N/A Have you written new tests for your changes, if applicable?

## Jetpack product discussion
None. This is an under-the-hood change with no user-visible impact.

## Does this pull request change what data or activity we track or use?
No.

## Testing instructions:
- Enable `wordpress.com` notifications on your test site
- Load a page on your test site with a `wordpress.com` user and ensure that `wordpress.com` notifications work on the admin bar.
